source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project-directory/__tests__/page.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  interactor,
+  Interactor,
+  text
+} from '@bigtest/interactor';
+
+import orgSelectInteractor from '@ui/components/inputs/organization-select/__tests__/page';
+import tableInteractor from '@ui/components/project-table/__tests__/page';
+
+class Directory {
+  header = text('[data-test-directory-header]');
```

Comment:
Same as the other comment we can have a better name for this

Reply:
but it _is_ the header

Reply:
ok